docs/fabric-native-components.md

```diff
@@ -128,7 +146,7 @@ The specification is used by the React Native's Codegen tools to generate platfo
      "type": "components",
      "jsSrcsDir": "specs",
      "android": {
-       "javaPackageName": "com.nativelocalstorage"
+       "javaPackageName": "com.webview"
```

Comment:
This seemed like an inaccuracy in the original post, but let me know if it should be `com.nativelocalstorage` for some reason. Thank you!

Reply:
nope it should be `com.webview` after we moved everything over to use WebViews